### PR TITLE
Swap underscore out for lodash

### PR DESCRIPTION
**Pull request by PeteJodo**

### CODE REVIEW

File: copy-text.js [hunk 1]
```diff
@@ -1,6 +1,8 @@
 "use strict";
 var serverVars = require('server-vars');
-var _ = require('underscore');
+var assign = require('lodash.assignin');
```
Comment (PeteJodo) [inline copy-text.js]: note that I'm using `lodash.assignin`. This seems to be the one that is equivalent to `_.extend`

Comment (antialias) [inline copy-text.js]: If you're using assignin, then call the variable `assignIn`

Comment (antialias) [inline copy-text.js]: Also, why is this necessary?

Comment (PeteJodo) [inline copy-text.js]: it is in the lodash docs. `extend` is deprecated for lodash and was renamed to `assignIn`

Comment (antialias) [inline copy-text.js]: Why is it necessary to get inherited properties here? I've been using `lodash.assign` whenever `Object.assign` isn't available.

Comment (PeteJodo) [inline copy-text.js]: Yeah you're right. I was only looking at the renaming in lodash docs. Should've thought about it a little more. Will fix

Comment (PeteJodo) [inline copy-text.js]: Fixed

**Comment by antialias** [comment]
please bump versino and publish
